src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderBookingRepository.java

```diff
@@ -30,6 +30,10 @@ public interface OffenderBookingRepository extends
 
     Optional<OffenderBooking> findByOffenderNomsIdAndBookingSequence(String nomsId, Integer bookingSequence);
 
+    default Optional<OffenderBooking> findLatestOffenderBookingByNomsId(String nomsId) {
+        return findByOffenderNomsIdAndBookingSequence(nomsId, 1);
```

Comment:
seemed silly to be passing `1` around when calling `findByOffenderNomsIdAndBookingSequence` so added new method